### PR TITLE
vmware-workstation-player-np: Add version 17.0.0-20800274

### LINKED ISSUE #87: [Request] VMware Workstation Player
### Criteria

For a package to be acceptable in the Nonportable bucket, it should be:

- [x] a reasonably well-known and widely used application
- [x] the latest stable version of the program
- [x] the full version i.e. not a trial version

### Information

**Name:** VMware Workstation Player

**Description:** VMware Workstation Player, formerly VMware Player, is a virtualization software package for x64 computers running Microsoft Windows or Linux, supplied free of charge by VMware, Inc.

**Homepage:** vmware.com

**Download link(s):**
https://www.vmware.com/go/getplayer-win 
https://customerconnect.vmware.com/en/downloads/details?downloadGroup=WKST-PLAYER-1624&productId=1039

**Some indication of popularity/repute:** https://github.com/ScoopInstaller/Extras/issues/6737

### PR DESCRIPTION
**Pull request by Zliced13**
Closes #87

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/vmware-workstation-player-np.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        "}"
+    ],
+    "checkver": {
+        "url": "https://scoop.sh",
+        "useragent": "$app/$version",
+        "script": [
+            "$redirUrl = [System.Net.HttpWebRequest]::Create('https://www.vmware.com/go/getplayer-win').GetResponse().ResponseUri.AbsoluteUri",
+            "return \"$redirUrl\""
+        ],
+        "regex": "WKST-PLAYER-(<CleanVer>[\\d]+)\\/VMware-player-full-([\\d.]+-[\\w]+)",
```
Comment (rasa) [inline bucket/vmware-workstation-player-np.json]: @Zliced13 I think a question mark needs to be added:
```
WKST-PLAYER-(?<CleanVer>[\\d]+)\\/VMware-player-full-([\\d.]+-[\\w]+)
```

Comment (kris3713) [reply bucket/vmware-workstation-player-np.json]: I'll try this later on today.

**Comment by Zliced13** [comment]
@rasa I need help with hash extraction.

**Comment by rasa** [comment]
Per
```bash
wget --no-show-progress -q -O - "https://customerconnect.vmware.com/channel/public/api/v1.0/dlg/details?locale=en_US&downloadGroup=WKST-PLAYER-1700" | jq -r .downloadFiles[0].sha256checksum
ae238c457a100f15a6d009bfeee4ff277dce181e96a279d486f07f30da433183
```

**Pull request merged**